nautobot/core/settings.py

```diff
@@ -124,7 +124,10 @@
 SANITIZER_PATTERNS = [
     # General removal of username-like and password-like tokens
     (re.compile(r"(https?://)?\S+\s*@", re.IGNORECASE), r"\1{replacement}@"),
-    (re.compile(r"(username|password|passwd|pwd)((?:\s+is.?|:)?\s+)\S+", re.IGNORECASE), r"\1\2{replacement}"),
+    (
+        re.compile(r"(username|password|passwd|pwd|secret|secrets)([\"']?(?:\s+is.?|:)?\s+)\S+[\"']?", re.IGNORECASE),
```

Comment:
Probably also a good idea to provide an example of what kind of cases this covers in a comment

Reply:
@gsnider2195  In a comment in the PR or in the doc string?

Reply:
Personally I'd vote for a comment here and maybe a note in the documentation as well.

Reply:
@jeffkala The docs update is the only thing remaining before we can merge this, I think?

Reply:
will get the docs added today.

Reply:
@glennmatthews fit best to put some additional docs in the optional-settings existing docs? Or create a new page of sorts?  Not seeing anywhere else that fits in the current docs structure.

Reply:
Yeah I was thinking in the optional-settings docs for SANITIZER_PATTERNS.

Reply:
Just updated optional settings with some examples.